cms/static/js/programs/models/api_config_model.js

```diff
@@ -0,0 +1,17 @@
+define([
+        'backbone'
+    ],
+    function( Backbone ) {
+        'use strict';
+
+        return Backbone.Model.extend({
+            defaults: {
+                username: '',
+                lmsBaseUrl: 'http://127.0.0.1:8001/',
+                programsApiUrl: 'http://127.0.0.1:8004/',
```

Comment:
I think it would be better to default these to null and cause the application to blow up if they aren't overridden. These defaults may work for local development in a very specific context, but will inevitably lead to strange or incorrect behavior.